bin/diagnostic.js

```diff
@@ -1 +1,16 @@
 'use strict';
+const ask = require('../lib/ask.js');
+
+let answer = 'not empty';
+
+while (answer !== '') {
+  answer = ask("Guess my secret? ");
+  if (answer < 1 && answer > 10) {
```

Comment:
This block never runs because it's not possible for answer to simultaneously be less than 1 and greater than 10.